src/frontend/src/views/Upload/components/Samples/components/AlertTable/index.tsx

```diff
@@ -19,6 +19,8 @@ const ERROR_CODE_MESSAGES: Record<ERROR_CODE, string> = {
   [ERROR_CODE.MISSING_FIELD]: "placeholder",
   [ERROR_CODE.OVER_MAX_SAMPLES]:
     "This file contains more than 500 samples, which exceeds the maximum for each upload process. Please limit the samples to 500 or less",
+  [ERROR_CODE.DUPLICATE_PUBLIC_IDS]: "placeholder",
```

Comment:
Do you want me to fill these parts in?

Reply:
i'll write a comment for this, we actually don't need this to be filled in for `AlertTable` as this was used in the upload flow when uploading samples from a fasta. I only added these here to get around the type errors that i was getting from not providing these values. Same thing is happening in `src/frontend/src/views/Upload/components/Metadata/components/ImportFile/components/Alerts/Error.tsx` where i added the two new errors with placeholder even though the Error component does not handle the duplicateIds case. (the duplicate IDs error is different enough that it doesn't make sense to share the default `Error` components)

Reply:
i was trying not to get into type hell by changing the ERRORCODE type, but if you think i should go that route then i can, otherwise i can add some comments to explain what's happening

Reply:
i'll change the type, update coming soon

Reply:
ok type updated, was not as bad as i initially thought